op_builder/builder.py

```diff
@@ -614,6 +614,9 @@ def builder(self):
         if not self.build_for_cpu and self.enable_bf16:
             compile_args['cxx'].append("-DBF16_AVAILABLE")
 
+        if self.is_rocm_pytorch():
```

Comment:
@rraminen - do we need both this change and the one in #4570?

Reply:
My understanding is that https://github.com/microsoft/DeepSpeed/pull/4585 is for non-JIT builds and https://github.com/microsoft/DeepSpeed/pull/4570 is for JIT builds. So both of them are required. Please correct me if wrong.